proxy/src/main/java/com/velocitypowered/proxy/connection/client/ClientConfigSessionHandler.java

```diff
@@ -182,6 +185,13 @@ public void exception(Throwable throwable) {
    * @return a future that completes when the config stage is finished
    */
   public CompletableFuture<Void> handleBackendFinishUpdate(VelocityServerConnection serverConn) {
+    String brand = serverConn.getPlayer().getClientBrand();
+    if (brand != null && brandChannel != null) {
+      PluginMessage brandPacket = new PluginMessage(
+              brandChannel, Unpooled.wrappedBuffer(brand.getBytes()));
```

Comment:
This should not be the raw bytes, it should be written as a utf string to the buffer

Reply:
Alright done